Wikipedia/Code/NotificationsCenterOnboardingView.swift

```diff
@@ -0,0 +1,173 @@
+import SwiftUI
```

Comment:
Pretty cool, right? I think this view was a perfect case for it. The hosting controller/dismiss action juggling is annoying, but so great that a view like this can be simplified down to a single compact file.